torchvision/models/googlenet.py

```diff
@@ -25,43 +25,6 @@
 _GoogLeNetOutputs = GoogLeNetOutputs
 
 
-def googlenet(pretrained: bool = False, progress: bool = True, **kwargs: Any) -> "GoogLeNet":
```

Comment:
Literally just moving it to the bottom of the file so that we can use proper typing.